onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -98,7 +98,7 @@ ORT_API_STATUS_IMPL(OrtCreateEnvWithCustomLogger, OrtLoggingFunction logging_fun
   std::unique_ptr<ISink> logger = std::make_unique<LoggingWrapper>(logging_function, logger_param);
   auto default_logging_manager = std::make_unique<LoggingManager>(std::move(logger),
                                                                   static_cast<Severity>(default_warning_level), false,
-                                                                  LoggingManager::InstanceType::Default,
+                                                                  LoggingManager::InstanceType::Temporal,
```

Comment:
@yuanbyu  Would you please review this change?  As you are currently working on the logging stuff.

Reply:
This change was made because Custom Loggers can not instantiate Default Logger again.

---
In reply to: [320574344](https://github.com/microsoft/onnxruntime/pull/1714#discussion_r320574344) [](ancestors = 320574344)

Reply:
You got the error because in your code the Environment object got created twice.  Yuan is solving the problem by combining the OrtEnv and onnxruntime::Environment  class into just one.

Reply:
That would be helpful, still custom logger should be created with Temporal setting, custom loggers are not default.

---
In reply to: [321018313](https://github.com/microsoft/onnxruntime/pull/1714#discussion_r321018313) [](ancestors = 321018313)

Reply:
I have reworked it as a standalone executable.

---
In reply to: [321006933](https://github.com/microsoft/onnxruntime/pull/1714#discussion_r321006933) [](ancestors = 321006933,320574344)

Reply:
I have reworked this as a standalone exe, so this problem is no longer relevant for this change.

---
In reply to: [321018313](https://github.com/microsoft/onnxruntime/pull/1714#discussion_r321018313) [](ancestors = 321018313)